crypto/include/internal/bn_int.h

```diff
@@ -64,6 +64,11 @@ void bn_set_static_words(BIGNUM *a, BN_ULONG *words, int size);
  */
 int bn_set_words(BIGNUM *a, BN_ULONG *words, int num_words);
 
+int bn_prime_checks_for_size_ex(int prime_size, int security_bits);
+int bn_generate_prime_ex2(BIGNUM *ret, int bits, int safe,
+                          const BIGNUM *add, const BIGNUM *rem, BN_GENCB *cb,
+                          int checks);
```

Comment:
Nit pick: I thought we reserved `_ex` for "upgraded" public functions...  I really see no need for that for internal functions.  Maybe it's better to use `_int` for a suffix?

Reply:
Not sure that `bn_generate_prime` should be "upgraded". It returns since stone age a pseudo prime that passed some tests: no small (first `NUMPRIMES`) factors, and some MR tests with a defined assurance level. As OpenSSL grows up and fulfills new secure requirements, the user expects that in the current version the assurance level is as state of the art. So I see in fact no need to make the function `bn_generate_prime_ex` scalable. Otherwise we could (should?) make scalable the upper bound for small factors division too.